dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
it's just a generator to keep the `Dockerfile` more readable and well formatted and make it easier to add things by just adding them to the Dockerfile.yml